images/receiveGpgKeys.sh

```diff
@@ -16,7 +16,7 @@ fi
 for keyID in $KEY_IDS; do
     for i in {1..10}; do
         echo "Try # $i"
-        gpg --batch --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "$keyID" || \
+        gpg --batch --keyserver hkps://keyserver.ubuntu.com:443 --recv-keys "$keyID" || \
```

Comment:
out of curiousity, did this resolve the issues you were seeing with the gpg keys when building images?

Reply:
yes! this one solved the issue for me.